lib/aggregation/accumulator/src/index.js

```diff
@@ -9,12 +9,13 @@ const mconfigcb = require('abacus-metering-config');
 const moment = require('abacus-moment');
 const oauth = require('abacus-oauth');
 const router = require('abacus-router');
-const seqid = require('abacus-seqid');
```

Comment:
When you removed `seqid` did you remove it from the `package.json`?

Reply:
You reinstated the accumulator configuration logic back in the accumulator component, hence the `seqid` is now again required in the `package.json` file. You need to recover it.